src/components/SelectionList/SearchTableHeader.tsx

```diff
@@ -32,6 +32,11 @@ function SearchTableHeader({data}: SearchTableHeaderProps) {
     return (
         <View style={[styles.ph5, styles.pb3]}>
             <View style={[styles.flex1, styles.flexRow, styles.gap3, styles.ph4]}>
+                <SearchTableHeaderColumn
+                    containerStyle={[StyleUtils.getSearchTableColumnStyles(CONST.SEARCH_TABLE_COLUMNS.RECEIPT)]}
+                    textStyle={StyleUtils.getTextOverflowStyle('clip')}
```

Comment:
Why do we explicitly set the text-overflow style to `clip`? Or maybe this was agreed somewhere
The default seems to be `ellipsis` as noticed for `merchant` and `category` below: 
![Screenshot 2024-05-17 at 5 38 07 PM](https://github.com/Expensify/App/assets/3004659/36d06e8c-04be-4959-9978-84a088177eb9)

Reply:
It was mentioned here that we should allow an overflow for receipts - https://github.com/Expensify/App/pull/41957#issuecomment-2106298558